web-application/src/main/static/app/shared/endpoints.service.ts

```diff
@@ -5,37 +5,40 @@ import {Endpoints} from "./endpoints";
 import {Endpoint} from "./endpoint";
 import "../rxjs-operators";
 
-/**
- * This service is the doorway to endpoint discovery.
- * It contacts the 'only' known relative endpoint and retrieves a list of 'unknown' mapped endpoints.
- */
 @Injectable()
 export class EndpointsService {
 
-    map: {[name: string]: Endpoint} = {};
+    map: {[name: string]: Endpoint} =
```

Comment:
This doesn't feel good. Have we replaced the endpoint resolution app with a hard coded map, just to run in a node server?